app/controllers/suppliers_controller.rb

```diff
@@ -58,6 +58,6 @@ def meta_title
   end
 
   def current_country
-    permitted_params[:country]&.downcase
+    permitted_params[:country]&.downcase || "england"
```

Comment:
Had to add this back in to get the location switcher to load on the England pages. Happy to use an alternative method if there's a better way of doing this!